Demo/HYPForms/HYPSampleCollectionViewController.m

```diff
@@ -60,9 +69,18 @@ - (HYPFormsCollectionViewDataSource *)dataSource
     if (_dataSource) return _dataSource;
 
     _dataSource = [[HYPFormsCollectionViewDataSource alloc] initWithCollectionView:self.collectionView
+                                                                            layout:self.layout
                                                                    andFormsManager:self.formsManager];
 
-    _dataSource.dataSource = self;
+    _dataSource.configureCellForIndexPath = ^(HYPFormField *field, UICollectionView *collectionView, NSIndexPath *indexPath) {
+        BOOL isImageCell = (field.type == HYPFormFieldTypeCustom && [field.typeString isEqual:@"image"]);
```

Comment:
Could you switch places with this line and line 77 so that the `isImageCell` is coupled to `if (isImageCell)`?